### ISSUE TITLE
When installing Newtonsoft.Json version changes

### ISSUE THREAD

**Issue by delpher** [unitycontainer/unity#136]
When installing Unity 5.0.0.1 it will replace latest version of Newtonsoft.Json library with 6.0.0.0

**Comment by ENikS** [comment]
As far as I can see it comes as a dependency to one of these:
```
  Installing Newtonsoft.Json 6.0.4.
  Installing Microsoft.AspNet.WebApi.Client 5.2.3.
  Installing Microsoft.AspNet.WebApi.Core 5.2.3.
  Installing Microsoft.AspNet.WebApi.WebHost 5.2.3.
  Installing Microsoft.AspNet.WebApi 5.2.3.
```
I am not sure what I could do about it. Neither one of unity projects depends on Newton.Json directly.

**Comment by ENikS** [comment]
On second thought, I see it now includes all these binaries within the package. Not good. I'll work on it.
You could use individual packages for now:
```
Unity.Abstractions.2.0.2
Unity.Container.5.0.1
...
```

**Comment by delpher** [comment]
But Newtonsoft.Json library included in .nupkg of Unity 5.0.0.1 version. And many other not related libraries:
![unity_5 0 0 1_package_contents](https://user-images.githubusercontent.com/2911209/31955104-09c0a350-b8f0-11e7-90c8-586e21261497.png)

**Comment by ENikS** [comment]
Yes, I got that now.

**Comment by ENikS** [comment]
Fixed in v5.0.2
